app/Http/Controllers/AccountController.php

```diff
@@ -113,10 +114,20 @@ public function edit()
 
         $notificationOptions = $user->notificationOptions->keyBy('name');
 
+        if (GithubUser::canAuthenticate()) {
+            $githubUsers = json_collection(
+                $user->githubUsers()->withGithubInfo()->get(),
```

Comment:
I'm not sure why I thought multiple github users could be good idea but there's no such thing and I don't think it needs to be supported. It's especially weird when there's actual user interface to link the account(s?).

Reply:
there are at least a few people that have used multiple GitHub accounts on osu-wiki for whatever reason. I agree it's not important but it was easy to support...

Reply:
how many are there? And what's the reason? That just seems confusing.

Reply:
very few. I don't know either lol. anyway looking at it again, yeah it does seem odd to be able to set multiple, I'll make the account page only support linking one